README.md

```diff
@@ -70,11 +71,11 @@ After specifying `"SFSafeSymbols"` as a dependency of the target in which you wa
 
 ### Accio
 
-Do the same configurations as for Swift PM, then run `accio update` instead of `swift package update`.
+Do the same configurations as for SwiftPM, then run `accio update` instead of `swift package update`.
 
 ### Carthage
 
-Make the following entry in your Cartfile:
+Make the following entry to your Cartfile:
```

Comment:
To unify things, let's also change "Make" to "Add"

Reply:
Fixed. :)